website_docs/markdown/getting-started.md

```diff
@@ -0,0 +1,356 @@
+    SimpleSpanProcessor(ConsoleSpanExporter())
+)
+
+app = flask.Flask(__name__)
+FlaskInstrumentor().instrument_app(app)
+RequestsInstrumentor().instrument()
+
+
+@app.route("/")
+def hello():
+    tracer = trace.get_tracer(__name__)
+    with tracer.start_as_current_span("example-request"):
+        requests.get("http://www.example.com")
+    return "hello"
+
+
+app.run(debug=True, port=5000)
+```
+
+Now run the script, hit the root URL ([http://localhost:5000/](http://localhost:5000/)) a few times, and watch your spans be emitted!
+
+```
+python flask_example.py
+```
+
+## Configure Your HTTP propagator (b3, Baggage)
+
+A major feature of distributed tracing is the ability to correlate a trace across
+multiple services. However, those services need to propagate information about a
+trace from one service to the other.
+
+To enable this propagation, OpenTelemetry has the concept of [propagators](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md),
+which provide a common method to encode and decode span information from a request and response, respectively.
+
+By default, `opentelemetry-python` is configured to use the [W3C Trace Context](https://www.w3.org/TR/trace-context/)
+and [W3C Baggage](https://www.w3.org/TR/baggage/) HTTP headers for HTTP requests, but you can configure it to leverage different propagators. Here’s
+an example using Zipkin’s [b3 propagation](https://github.com/openzipkin/b3-propagation):
+
+```
+pip install opentelemetry-propagator-b3
+```
+
+Following the installation of the package containing the b3 propagator, configure the propagator as follows:
+
+```
+from opentelemetry.propagate import set_global_textmap
+from opentelemetry.propagators.b3 import B3Format
+
+set_global_textmap(B3Format())
+```
+
+## Use the OpenTelemetry Collector for traces
+
+Although it’s possible to directly export your telemetry data to specific backends, you might have more complex use cases such as the following:
+
+
+* A single telemetry sink shared by multiple services, to reduce overhead of switching exporters.
+
+
+* Aggregating traces across multiple services, running on multiple hosts.
+
+To enable a broad range of aggregation strategies, OpenTelemetry provides the [opentelemetry-collector](https://github.com/open-telemetry/opentelemetry-collector).
+The Collector is a flexible application that can consume trace data and export to multiple other backends, including to another instance of the Collector.
+
+Start the Collector locally to see how the Collector works in practice. Write the following file:
+
+```
+# /tmp/otel-collector-config.yaml
+receivers:
+    opencensus:
+        endpoint: 0.0.0.0:55678
+exporters:
+    logging:
+        loglevel: debug
+processors:
+    batch:
+    queued_retry:
+service:
+    pipelines:
+        traces:
+            receivers: [opencensus]
+            exporters: [logging]
+            processors: [batch, queued_retry]
+```
```

Comment:
collector should enable OTLP receiver

Reply:
good point, i've also updated the port :D